src/encoded/static/components/item-pages/FileView.js

```diff
@@ -0,0 +1,173 @@
+
+// Page containing the details of Items of type File
+export default class FileOverview extends DefaultItemView {
+    getTabViewContents() {
+        const initTabs = [];
+        initTabs.push(FileView.getTabObject(this.props));
+        return initTabs.concat(this.getCommonTabs()); // Add remainder of common tabs (Details, Attribution)
+    }
+}
+
+/**
+ * Decomposes the current file item's [href] and provides an onClick function
+ * to [children] for opening the file's raw JSON representation in a new page.
+ * @param {string} href the href of the file item being viewed
+ * @returns {JSX.Element}
+ */
+function ViewJSONAction({ href, children }) {
+    const urlParts = _.clone(memoizedUrlParse(href));
+    urlParts.search =
+        '?' +
+        queryString.stringify(_.extend({}, urlParts.query, { format: 'json' }));
+    const viewUrl = url.format(urlParts);
+    const onClick = (e) => {
+        if (window && window.open) {
+            e.preventDefault();
+            window.open(
+                viewUrl,
+                'window',
+                'toolbar=no, menubar=no, resizable=yes, status=no, top=10, width=400'
+            );
+        }
+    };
+    return React.cloneElement(children, { onClick });
+}
+
+// File Overview's header component containing breadcrumbs and filename
+const FileViewTitle = (props) => {
+    const { context } = props;
+
+    const breadcrumbs = [
+        { display_title: 'Home', href: '/' },
+        { display_title: 'Data' },
+        { display_title: 'Bechmarking Data' },
+        { display_title: context?.dataset?.toUpperCase() || '' },
+    ];
+
+    return (
+        <div className="file-view-title container-wide">
+            <nav className="file-view-title-navigation">
+                <ul className="breadcrumb-list">
+                    {breadcrumbs.map(({ display_title, href }, i, arr) => {
+                        return (
+                            <li className="breadcrumb-list-item" key={i}>
+                                <a
+                                    className={
+                                        'breadcrumb-list-item-link' +
+                                        (href ? '' : ' no-link')
+                                    }
+                                    href={href}>
+                                    {display_title}
+                                </a>
+                                {i < arr.length - 1 ? (
+                                    <i className="icon icon-fw icon-angle-right fas"></i>
+                                ) : null}
+                            </li>
+                        );
+                    })}
+                </ul>
+            </nav>
+            <h1 className="file-view-title-text">{context.display_title}</h1>
+        </div>
+    );
+};
+
+// Header component containing high-level information for the file item
+const FileViewHeader = (props) => {
+    const { context, session } = props;
```

Comment:
Might be good to do `context = {}` just so it has a default value and the line with selectedFile doesn't error out when indexing into context if it's not present. I don't think this should happen... but good to be safe.